src/main/java/FileUtility.java

```diff
@@ -0,0 +1,76 @@
+import java.io.*;
+import java.util.Scanner;
+
+public class FileUtility {
+    private double[] results;
+    private String[] wordsInLine;
+
+    public void readFile(String fileName) throws FileNotFoundException {
+        File file = new File(fileName);
+        Scanner scanner = new Scanner(file);
```

Comment:
Pamiętaj o zamykaniu pliku, albo najlepiej wprowadź try-with-resources https://youtu.be/LRwSHA9YDgY